ts/Core/Axis/Tick.ts

```diff
@@ -432,7 +432,7 @@ class Tick {
             // (#8809)
             if (
                 label.textWidth &&
-                !(labelOptions.style && labelOptions.style.width) &&
+                !labelOptions.style.width &&
                 !(label.styles as any).width
```

Comment:
All the `style` types are the one thing I wonder, if we should go with the built-in variant `CSSStyleDeclaration` of `lib.dom.d.ts`.

Reply:
Yes maybe, but I think it's a bit out of scope for this PR